README.md

```diff
@@ -145,7 +133,7 @@ For more information on the package content, visit [readthedocs](https://eo-lear
 
 ## More Examples
 
-Examples and introductions to the package can be found [here](https://github.com/sentinel-hub/eo-learn/tree/master/examples). A large collection of examples is available at the [`eo-learn-examples`](https://github.com/sentinel-hub/eo-learn-examples) repository. While the examples there are not always up-to-date they can be a great source of ideas.
+Examples and introductions to the package can be found [here](https://github.com/sentinel-hub/eo-learn/tree/master/examples). A larger collection of examples is available at the [`eo-learn-examples`](https://github.com/sentinel-hub/eo-learn-examples) repository. While the examples there are not always up-to-date they can be a great source of ideas.
```

Comment:
should we add somewhere that some tasks are also there?

Reply:
I'm not sure what you mean. To specifically mention that older eo-learn tasks were moved there? I can add that, yes.

Reply:
yes

Reply:
added